tests/MongoDB.Driver.Core.Tests/Specifications/server-selection/ServerSelectionTestRunner.cs

```diff
@@ -36,8 +34,12 @@ public class ServerSelectionTestRunner
 
         [Theory]
         [ClassData(typeof(TestCaseFactory))]
-        public void RunTestDefinition(BsonDocument definition)
+        public void RunTestDefinition(JsonDrivenTestCase testCase)
         {
+            var definition = testCase.Test;
+
+            JsonDrivenHelper.EnsureAllFieldsAreValid(definition, "_path", "in_latency_window", "operation", "read_preference", "suitable_servers", "topology_description", "heartbeatFrequencyMS", "error");
```

Comment:
NOTE: these changed are made just to be sure that no new test fields were added in the new tests.